sway/desktop/transaction.c

```diff
@@ -291,6 +291,12 @@ static void arrange_children(enum sway_container_layout layout, list_t *children
 		int width, int height, int gaps) {
 	int title_bar_height = container_titlebar_height();
 
+	bool child_titlebar = title_bar_height == 0;
+	if (active && active->current.grouped_titlebar_follows_border && active->current.border != B_NORMAL) {
+		child_titlebar = true;
+		title_bar_height = 0;
```

Comment:
We are setting `title_bar_height` to 0 here, why not keep the existing logic to check the variable == 0 instead of adding a bool?

Reply:
Yeah, good point - had some more complicated logic there temporarily and had missed that I had simplified it all the way back to just `title_bar_height == 0`.